guide/payments/introduction.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: guide
+title: Payments
+description: Diving into the user experience of moving Bitcoin
+nav_order: 4
+has_children: true
+permalink: /guide/payments/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+   image = "/assets/images/guide/payments/introduction.svg"
+   mobile = "/assets/images/guide/payments/introduction-mobile.svg"
+   alt-text = "A network of payment channels"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Payments and transactions
+
+At its core, the Bitcoin protocol enables transactions between people without intermediaries, regardless of where in the world they are.
+
+This chapter is meant to give an overview of bitcoin transactions, including sending, receiving, and transaction privacy, with advice and best practices. After this chapter, you should be able to design payment solutions that are well suited to your product's use case and understand best-practices for its implementation.
+
+We have already established that [Bitcoin is money](https://bitcoin.design/guide/getting-started/why-bitcoin-is-unique/#its-money-but-digital), so let's dive into how it can be used to facilitate payments.
+
+---
+
+#### Transactions overview (coming soon)
```

Comment:
Could you please use bold text instead of headers for consistency with the other section landing pages?

Reply:
sure, just fixed